hnn_core/cell.py

```diff
@@ -98,10 +98,6 @@ def create_soma(self):
         self.soma.Ra = soma_props['Ra']
         self.soma.cm = soma_props['cm']
 
-    def get_sections(self):
-        """Get sections."""
-        return [self.soma]
```

Comment:
I moved this to `basket.py` because it was making the code silently fail. When I removed `get_sections` in `pyramidal.py` (not aware that it was used somewhere), it started using this method which was inherited ...

Reply:
@jasmainak so `pyramidal.py` and `basket.py` each have their own `get_sections()` methods that shouldn't be unified?

Reply:
I guess "sections" refers to the different compartments a cell can have. A basket cell is modeled as a soma-only cell, whereas pyramidals have dendrites. I'm +1 for just deleting the method here, alternatively raising NotImplementedError?

Reply:
Yep, I was thinking the same. I will add it to the baseclass with a NotImplementedError :)

Reply:
> each have their own get_sections() methods that shouldn't be unified?

for now, I would say yes. Every cell type should define its own get_sections